### ISSUE TITLE
Support for latest AVX512, VNNI , etc in MKL version of Serving

### ISSUE THREAD

**Issue by whatdhack** [tensorflow/serving#1559]
### Describe the problem the feature is intended to solve
Currently the MKL build is not taking advantage of acceleration  for neural net  that are available in Xeon CPUs - notably VNNI.  VNNI supports faster FP16, BF16, and INT8 inference. 

### Describe the solution
Option in Docker.mkl to enable VNNI ? 

### Describe alternatives you've considered
Use MKL without VNNI. 

### Additional context
N/A

**Comment by ganand1** [comment]
Please use 
https://github.com/tensorflow/serving/blob/master/.bazelrc#L24

--config=nativeopt while building on machines where VNNI instruction is supported. 


https://github.com/tensorflow/serving/blob/master/tensorflow_serving/tools/docker/Dockerfile.devel-mkl#L97

Passing in TF_SERVING_BUILD_OPTIONS=--config=nativeopt  while building using above docker

**Comment by pindinagesh** [comment]
Hi @whatdhack 

Could you kindly confirm on the comment from @ganand1 and suggest whether this can be closed.Thanks.

**Comment by pindinagesh** [comment]
Hi @whatdhack 

Could you please look at the [documentation](https://github.com/tensorflow/serving/tree/master/tensorflow_serving/tools/docker) about MKL version of Serving and Closing this issue due to inactivity. Please feel free to reopen if this still exist. Thanks.
